package.json

```diff
@@ -8,8 +8,10 @@
     "build:prod": "npm run build -- --public-path='https://cdn.ampproject.org/sw/'",
     "transpile": "tsc -p ./src/tsconfig.json",
     "pretest": "npm run transpile && npm run build -- --public-path='/test/dist/' && mkdir -p test/dist && cp -R dist/* test/dist",
-    "test": "node -r esm ./test/index.js",
-    "posttest": "npm run test:perf",
+    "test": "yarn test:unit && yarn test:e2e && yarn test:perf",
```

Comment:
Let's not use `&&` since this won't work on Windows systems. Instead `npm-run-all` can be used across platforms.